hikari/impl/special_endpoints.py

```diff
@@ -1165,7 +1181,6 @@ async def create(
             self._name,
             self._description,
             guild=guild,
-            default_permission=self._default_permission,
```

Comment:
Don't you need to pass through `default_member_permissions` and `dm_enabled` here?